### ISSUE TITLE
How long to train a good model?

### ISSUE THREAD

**Issue by guvcolie** [yifjiang/relative-depth-using-pytorch#4]
Thank you for your excellent code!
How long to get a good model based on DIW from scratch? I found the loss decrease very slowly...

**Comment by yifjiang** [comment]
About 200000 batches would suffice.
